stac-check.config.yml

```diff
@@ -0,0 +1,27 @@
+linting:
+  # Identifiers should consist of only lowercase characters, numbers, '_', and '-'
+  searchable_identifiers: true
+  # Item name '{self.object_id}' should not contain ':' or '/'
+  percent_encoded: true
+  # Item file names should match their ids
+  item_id_file_name: true
+  # Collections and catalogs should be named collection.json and catalog.json
+  catalog_id_file_name: true
+  # A STAC collection should contain a summaries field
+  check_summaries: true
+  # Datetime fields should not be set to null
+  null_datetime: true
```

Comment:
IMO this should default to `False`, as it is perfectly valid to have a `null` datetime if `start_datetime` and `end_datetime` are set.

Reply:
This a warning - like a recommendation - I could explain it better 
```The specification does allow one to set the datetime field to null, but it is strongly recommended to populate the single datetime field, as that is what many clients will search on. If it is at all possible to pick a nominal or representative datetime then that should be used.```

Reply:
https://github.com/radiantearth/stac-spec/blob/master/best-practices.md#datetime-selection

Reply:
Hmm yeah good point -- I think my quibble is with the best practices text instead then :-). Disregard my comment.

Reply:
Haha yea